package.json

```diff
@@ -46,7 +45,6 @@
     "build": "babel src --out-dir .",
     "check": "node_modules/.bin/eslint --ext \".js\" --ignore-pattern node_modules --quiet .",
     "test": "NODE_ENV=test mocha --opts test/mocha.opts",
-    "test-no-mock": "TEST_SERVER=1 NOCK_OFF=true NODE_ENV=test mocha --opts test/mocha.opts",
-    "postinstall": "npm run build"
```

Comment:
Without removing postinstall, `npm install` would still try to build with babel even when installing a built version